src/components/summary-list/index.md.njk

```diff
@@ -58,6 +59,50 @@ If your summary list does not have any actions, you can choose to remove the sep
 
 To remove borders on a single row, use the `govuk-summary-list__row--no-border` class.
 
+## Showing multiple summary lists
+
+If you’re showing multiple summary lists on a page, you can show each list within a summary card. This lets you visually separate them and give each list a title and some actions.
+
+Use summary cards when you need to:
+- show multiple summary lists that are related to one another
+- show actions that will apply to everything in a summary list
+
+But do not use summary cards if you only need to show a small amount of related information. Use summary lists instead, and structure them with headings if needed.
+
+If you’re showing summary cards at the end of a journey, you might want to familiarise the user with them earlier on — such as when the user reviews individual sections.
+
+### Summary card with a title
+
+Use the summary card’s header to give each summary list a title.
+
+Each title should be unique and help identify what the information in the summary list is meant to describe. For example, this could be a person, organisation or professional qualification.
+
+A title can simply be made up of one or two important values in the summary list — such as the first and last name of a person. Try to keep titles short.
+
+{{ example({group: "components", item: "summary-list", example: "card-with-title", html: true, nunjucks: true, open: false}) }}
+
+### Summary card with a title and actions
+
+You can also add action links in the header. These should be actions that apply to all values in the summary list, such as ‘Delete’, ‘Edit’ or ‘Update status’. 
+
+For example, you might choose to show a single ‘Change’ link in the header, instead of repeating change links for every row, if they’d all go to the same page anyway.
+
+Do not add more than 3 actions in a header.
+
+Remember to follow the rest of the guidance in ‘Adding actions to each row’ to help assistive technology users.
+
+{{ example({group: "components", item: "summary-list", example: "card-with-actions", html: true, nunjucks: true, open: false}) }}
+
+### Summary card without a title
+
+Summary cards should have a title — but there are some instances when you might choose not to add a title.
+
+For example, you might need to show multiple summary lists and visually separate them — but it does not make sense to write titles and headings.
+
+Similarly, you might need to show a mix of summary lists that do, and do not, call for titles.
+
+{{ example({group: "components", item: "summary-list", example: "card-no-title", html: true, nunjucks: true, open: false}) }}
+
 ## Research on this component
 
 This component was developed and tested by the Government Digital Services as part of the [check answers pattern](/patterns/check-answers/).
```

Comment:
```suggestion
This component was developed and tested by the Government Digital Services as part of the [check answers pattern](/patterns/check-answers/).

The Department for Education contributed the summary card. It’s being used on:

* apply for teacher training (used by citizens)
* register trainee teachers (used by teacher training providers)

It is also being used by other departments:

* manage supervisions (Ministry of Justice)
* submit social housing lettings and sales data (Department for Levelling Up, Housing & Communities)
```

Reply:
Reckon this one with bullet points reads better of the 2 options. I've removed an extra 's' at the end of GDS and added a few words to lead-in the lists.

```suggestion
This component was developed and tested by the Government Digital Service as part of the [check answers pattern](/patterns/check-answers/).

The Department for Education contributed the summary card. It’s being used on services to:
- apply for teacher training (used by citizens)
- register trainee teachers (used by teacher training providers)

It is also being used by other departments to help users:
- manage supervisions (Ministry of Justice)
- submit social housing lettings and sales data (Department for Levelling Up, Housing & Communities)
```

Reply:
Cool. Feel free to merge.